docs/naps/2-conda-based-packaging.md

```diff
@@ -0,0 +1,609 @@
+(nap2)=
+
+# NAP-2 — Distributing napari with conda-based packaging
+
+```{eval-rst}
+:Author: Jaime Rodríguez-Guerra, Gonzalo Peña-Castellanos
+:Created: 2022-05-05
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+:Resolved: <date resolved, in yyyy-mm-dd format>
+:Status: Draft
+:Type: <Standards Track | Process>
+:Version effective: <version-number> (for accepted NAPs)
+```
+
+## Abstract
+
+napari can be installed through different means, including `pip` and `conda`. However, users
+not exposed to package managers and command-line interfaces might prefer other options more
+native to their operating system of choice. This is usually achieved through graphical
+installers with a step-by-step interface.
+
+This NAP discusses how we will use and adapt tools borrowed from the `conda` packaging
+ecosystem to build platform-specific installers and implement update strategies for napari and
+its plugin ecosystem.
+
+## Motivation and Scope
+
+napari is packaged for PyPI [^pypi-napari] and conda-forge [^napari-feedstock] and can be
+installed with any client supporting these repositories (e.g., `pip` and `conda`,
+respectively). For a number of releases, platform-specific installers were also provided with
+Briefcase [^briefcase].
+
+Briefcase relies on PyPI packaging to assemble the installer. The PyPI packaging strategy,
+however, presents a series of limitations for the napari ecosystem:
+
+* No standardized building infrastructure. PyPI accepts submissions from any user without
+  requiring any validation or review. As a result, packages can be built using arbitrary
+  toolchains or expecting different libraries in the system. `cibuildwheel` [^cibuildwheel]
```

Comment:
It may be worth adding information about: 

* audithwheel: for collecting dependencies in linux https://github.com/pypa/auditwheel
* delocate: same for macos  https://github.com/matthew-brett/delocate
* delvewheel: same for windows, but experimental https://github.com/adang1345/delvewheel  

As they are real packages that solving problem of system installed libraries. cibuildwheel is only workflow wrapper.

Reply:
Thanks! Added them as footnotes.